pandas/tests/frame/test_timeseries.py

```diff
@@ -143,7 +143,7 @@ def test_pct_change_shift_over_nas(self):
 
         df = DataFrame({'a': s, 'b': s})
 
-        chg = df.pct_change()
+        chg = df.pct_change(fill_method='ffill')
```

Comment:
Why do we need to add this now? API change?

Reply:
Yes, default is skipna=True, as for the rest of pandas methods. See discussion with @jreback in `pandas/core/generic.py`.